src/twisted/conch/test/test_channel.py

```diff
@@ -93,13 +94,12 @@ def connectSSHTransport(service, hostAddress=None, peerAddress=None):
 
 
 
-class ChannelTests(unittest.TestCase):
+@skipIf(skipTest, skipTest)
```

Comment:
Since we are looking at strict typing... should this be something like this?
With an explicit bool type and not with str.
```suggestion
@skipIf(bool(skipTest), skipTest)
```

Reply:
The first argument to skipIf takes a condition which is internally evaluated:
https://github.com/python/cpython/blob/3.8/Lib/unittest/case.py#L132

so I think it is OK to leave it as-is without an explicit bool type.

Reply:
I don't think wrapping in `bool` adds any value here if it's not causing a typing error.
If `skipIf` is not type hinted yet, when it gets a type hint, this will raise an error if a bool is necessary, and will not if it isn't.

Reply:
ok. I am still not sure what is the purpose of all this type checking in Python :)

I was thinking that if the direction of Twisted is to remove dynamic typing, then any change to the existing code should be done with strict types and for example don't mix an empty string with boot or 0 .